modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/view_object_entries.jsp

```diff
@@ -18,17 +18,32 @@
 
 <%
 ViewObjectEntriesDisplayContext viewObjectEntriesDisplayContext = (ViewObjectEntriesDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+boolean isObjectWidget = viewObjectEntriesDisplayContext.isObjectWidget();
```

Comment:
@danvb112  can we suppress this variable and put `iewObjectEntriesDisplayContext.isObjectWidget()` directly inside the `when` clause? thx